index.html

```diff
@@ -66,7 +70,7 @@ <h3>2015.10.15(THU) ~ 16(FRI)</h3>
         </div>
     </section>
     <!-- start of main program -->
-    <section class="program">
+    <section class="program" id="program-section">
         <div class="a">
             <div class="main-program">
                 <h3>Main Program</h3>
```

Comment:
- [ ] As I mentioned above :point_up: please follow the Behance design as much as possible if you check your design there are some spacing and text alignment problems in the mobile version of the main program section so kindly fix these issues as well :+1: check below point_down the comparison between your design and Behance's design

**Your Design**
![Screenshot from 2023-05-30 00-56-39](https://github.com/sadaf-Daneshgar/Capstone-Project/assets/45226848/227eab6d-f983-4b56-b55f-53979d4cdfd3)

**Behance Design**
![Screenshot from 2023-05-30 00-57-07](https://github.com/sadaf-Daneshgar/Capstone-Project/assets/45226848/09fcc535-5963-40d6-b93f-790adeff0b7b)